launchSettings.json

```
@@ -0,0 +1,22 @@
+{
+  "profiles": {
+    "Web Profiel": {
+      "commandName": "DockerCompose",
+      "commandVersion": "1.0",
```

Hiermee kan je web openen in de browser
```suggestion
      "commandVersion": "1.0",
      "composeLaunchServiceName": "website",
      "composeLaunchUrl": "{Scheme}://localhost:{ServicePort}/swagger",
```